src/middlewared/middlewared/plugins/truecommand/wireguard.py

```diff
@@ -135,5 +137,6 @@ async def start_truecommand_service(self):
 
     @private
     async def stop_truecommand_service(self):
+        await self.middleware.call('service.reload', 'http')
```

Comment:
Dont you need to do that after you stop truecommand?

Reply:
At this point, TC connection status has already shifted, so it's safe to do it here. We can do afterwards as well but it won't change much ?